x-pack/platform/plugins/private/reporting/public/management/mount_management_section.tsx

```diff
@@ -21,49 +21,92 @@ import {
   ReportingAPIClient,
   KibanaContext,
 } from '@kbn/reporting-public';
-import { ReportListing } from '.';
+import { ActionsPublicPluginSetup } from '@kbn/actions-plugin/public';
+import { QueryClientProvider } from '@tanstack/react-query';
+import { EuiLoadingSpinner } from '@elastic/eui';
+import { Route, Router, Routes } from '@kbn/shared-ux-router';
+import { Redirect } from 'react-router-dom';
+import { queryClient } from '../query_client';
+import { Section } from '../constants';
 import { PolicyStatusContextProvider } from '../lib/default_status_context';
 
-export async function mountManagementSection(
-  coreStart: CoreStart,
-  license$: LicensingPluginStart['license$'],
-  dataService: DataPublicPluginStart,
-  shareService: SharePluginStart,
-  config: ClientConfigType,
-  apiClient: ReportingAPIClient,
-  params: ManagementAppMountParams
-) {
+const ReportingTabs = lazy(() => import('./components/reporting_tabs'));
+
+export async function mountManagementSection({
+  coreStart,
+  license$,
+  dataService,
+  shareService,
+  config,
+  apiClient,
+  params,
+  actionsService,
+  notificationsService,
+}: {
+  coreStart: CoreStart;
+  license$: LicensingPluginStart['license$'];
+  dataService: DataPublicPluginStart;
+  shareService: SharePluginStart;
+  config: ClientConfigType;
+  apiClient: ReportingAPIClient;
+  params: ManagementAppMountParams;
+  actionsService: ActionsPublicPluginSetup;
+  notificationsService: NotificationsStart;
+}) {
   const services: KibanaContext = {
     http: coreStart.http,
     application: coreStart.application,
+    settings: coreStart.settings,
     uiSettings: coreStart.uiSettings,
     docLinks: coreStart.docLinks,
     data: dataService,
     share: shareService,
+    actions: actionsService,
+    notifications: notificationsService,
```

Comment:
Based on some of my comments about leveraging the `useKibana` hook, you might want to add `toasts` and `userProfile` to this set of services.